js/packages/cli/README.md

```diff
@@ -64,10 +115,9 @@ metaplex mint_one_token -e devnet --keypair /path/to/user-payer
     "category": "video",
     "creators": [
       {
-        "address": "SOLFLR15asd9d21325bsadythp547912501b",
```

Comment:
This address is incorrect solana adrs and it breaks the upload fn w/o explaining why